tests/mappers/test_action_mapper.py

```diff
@@ -30,7 +30,7 @@
 TEST_DAG_NAME = "dag_name"
 
 
-class TestMapper(ActionMapper):
+class AMapper(ActionMapper):
```

Comment:
```suggestion
class ActionMapperTestImpl(ActionMapper):
```
or similar, somehow `AMapper` bothers me.

Reply:
Or can there be no `Test` word anywhere in the name? Not even in the middle?

Reply:
Yeah. Will change. It's Test prefix in class name (and class has to be in test_*.py or *_test.py in this case):

Pytest has several criteria to determine if class is a Test class. From https://docs.pytest.org/en/latest/goodpractices.html#conventions-for-python-test-discovery

```
pytest implements the following standard test discovery:

If no arguments are specified then collection starts from testpaths (if configured) or the current directory. Alternatively, command line arguments can be used in any combination of directories, file names or node ids.
Recurse into directories, unless they match norecursedirs.
In those directories, search for test_*.py or *_test.py files, imported by their test package name.
From those files, collect test items:
test prefixed test functions or methods outside of class
test prefixed test functions or methods inside Test prefixed test classes (without an __init__ method)
For examples of how to customize your test discovery Changing standard (Python) test discovery.

Within Python modules, pytest also discovers tests using the standard unittest.TestCase subclassing technique.


```